### ISSUE TITLE
the source code of the TSI architecture

### ISSUE THREAD

**Issue by KevenLee** [omtcyang/TSI-CN-dataset#1]
Is it convenient to share the code for the article 'Traffic Sign Interpretation in Real Road Scene', or is there a plan to share the code

**Comment by omtcyang** [comment]
yes, we are re-writing the code and will share it soon, please keep following this repo.

**Comment by KevenLee** [comment]
I can't wait any longer, I'm here to hurry up the update. When will the code be updated?

**Comment by omtcyang** [comment]
we release the dataset on GitHub already. And we prepare an end to end vlm based model now, please keep following us

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: KevenLee ***@***.***>
发送时间: Wednesday, September 4, 2024 3:37:25 PM
收件人: omtcyang/TSI-CN_Traffic-Sign-Interpretation-via-Natural-Language-Description- ***@***.***>
抄送: omtcyang ***@***.***>; Comment ***@***.***>
主题: Re: [omtcyang/TSI-CN_Traffic-Sign-Interpretation-via-Natural-Language-Description-] the source code of the TSI architecture (Issue #1)


I can't wait any longer, I'm here to hurry up the update. When will the code be updated?

―
